README.md

```diff
@@ -129,17 +129,21 @@ letsencrypt::certonly { 'foo':
 }
 ```
 
-To automatically renew a certificate, you can pass the `manage_cron` parameter.
-You can optionally add a shell command to be run on success using the `cron_success_command` parameter.
-You can optionally add a shell command to be run on before using the `cron_before_command` parameter.
-You can optionally specify one or multiple days of the month when to run the cron using the `cron_monthday` parameter (default is every day).
+To automatically renew a certificate, you can pass the `manage_cron` parameter.  
```

Comment:
Could you remove the trailing whitespace here?

Reply:
Added to for newlines when renderd

Reply:
A lot of scripts and editors automatically strip trailing whitespace and it's also very hard to review. It's probably better to rewrite as a list:
* `manage_cron` can be used to automatically renew the certificate
* `cron_success_command` can be used to run a shell command on a successful renewal
* `cron_before_command` can be used to run a shell command before a renewal
* ....